contracts/REXMarket.sol

```diff
@@ -46,7 +47,9 @@ abstract contract REXMarket is Ownable, SuperAppBase, Initializable {
     }
 
     struct OracleInfo {
-        uint256 requestId;
+        address pool;
+        address inputToken;
+        address outputToken;
         uint256 usdPrice;
         uint256 lastUpdatedAt;
```

Comment:
We don't need to store usdPrice and lastUpdated at anymore I don't think, the `updatePrices` methods are a burden to call all the time, the oracle query should be "just in time" for the swap, and it doesn't need to persist the price or last updated at

Reply:
Cool, I agree on this. As I earlier thought this kept cause of gas issues